src/app/pages/login/login.component.scss

```diff
@@ -80,6 +81,28 @@
   color: #222222;
 }
 
+.msw-login-header {
+  margin-top: 36px;
+  margin-right: 48px;
+  font-size: 21px;
+  letter-spacing: 0.4px;
+  color: #222222;
+  text-align: right;
+  line-height: 80px;
+}
+
+.msw-login-footer {
+  margin-top: 36px;
+  vertical-align: baseline;
+  //  font-size: 21px;
```

Comment:
Can you delete the comment? (we have source control...)

Reply:
Done